definitions/immuno.wdl

```diff
@@ -689,38 +699,25 @@ workflow immuno {
       hlaConsensus.hla_call_files
     ])
 
-    # --------- FDA metrics outputs ------------------------------------
-
-    FdaMetricBundle fda_metrics = object {
-      unaligned_normal_dna: generateFdaMetrics.unaligned_normal_dna_metrics,
-      unaligned_tumor_dna: generateFdaMetrics.unaligned_tumor_dna_metrics,
-      unaligned_tumor_rna: generateFdaMetrics.unaligned_tumor_rna_metrics,
-      aligned_normal_dna: generateFdaMetrics.aligned_normal_dna_metrics,
-      aligned_tumor_dna: generateFdaMetrics.aligned_tumor_dna_metrics,
-      aligned_tumor_rna: generateFdaMetrics.aligned_tumor_rna_metrics
-    }
 
     # --------- Other Outputs ------------------------------------------
 
-    MHC pvactools = object {
+    MHC pVACseq = object {
       mhc_i: pvacseq.mhc_i,
       mhc_ii: pvacseq.mhc_ii,
       combined: pvacseq.combined,
       phase_vcf: [phaseVcf.phased_vcf, phaseVcf.phased_vcf_tbi]
     }
 
-    MHC pvacfuse_predictions = object {
+    MHC pVACfuse = object {
       mhc_i: pvacfuse.mhc_i,
       mhc_ii: pvacfuse.mhc_ii,
       combined: pvacfuse.combined
     }
 
     File pvacseq_annotated_expression_vcf_gz = pvacseq.annotated_vcf
     File pvacseq_annotated_expression_vcf_gz_tbi = pvacseq.annotated_vcf_tbi
-    File pvacseq_annotated_variants_tsv = pvacseq.annotated_tsv
+    File variants_final_annotated_tsv = pvacseq.annotated_tsv
```

Comment:
Note that this won't change the name of this file.  To do that you'd need to change what value is passed to this input:
https://github.com/wustl-oncology/analysis-wdls/blob/90c4ff39c23cd432187108399e5f0996741c6885/definitions/tools/add_vep_fields_to_table.wdl#L8
Right now that value is set here:
https://github.com/wustl-oncology/analysis-wdls/blob/90c4ff39c23cd432187108399e5f0996741c6885/definitions/subworkflows/pvacseq.wdl#L159
so it might need to be exposed as an input up to the higher level so our immuno workflow change set the value as desired.